src/main/java/me/lemire/longcompression/differential/LongDelta.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * This code is released under the
+ * Apache License Version 2.0 http://www.apache.org/licenses/.
+ *
+ * (c) Daniel Lemire, http://lemire.me/en/
+ */
+
+package me.lemire.longcompression.differential;
+
+/**
+ * Generic class to compute differential coding.
+ * 
+ * @author Benoit lacelle
+ * 
+ */
+public final class LongDelta {
```

Comment:
Introduced but not tested (just duplicated and adjusted for longs)

Reply:
I am sure we can use more testing, but the code looks fine and it appears to run in the tests.